packages/gatsby-cli/src/structured-errors/error-map.js

```diff
@@ -47,7 +47,9 @@ const errorMap = {
         ? `\n\nDid you mean to use ` + `"${context.closestFragment}"?`
         : ``
 
-      return `There was an error in your GraphQL query:\n\nThe fragment "${context.fragmentName}" does not exist.${closetFragment}`
+      return `There was an error in your GraphQL query:\n\nThe fragment "${
```

Comment:
Not sure why my merge with master reformatted all of these. Maybe it's something with a local prettier config? 🤔